Wed/067_114_plagarism_detection_from_set_of_files.txt

```diff
@@ -3,11 +3,21 @@ CE067
 CE114
 
 Names:
+	Meet Mehta:CE-067
+	Jay Shah:CE-114 
 
-Project title:
+Project title:Plagarism Detection System 
```

Comment:
Make title more specific to reflect the scope of your project. Like it is not generic plagarism detection system, but it will try to find plagarism from set of source code files. Give it a title which reflects its scope.